test/e2e/fixtures/ngJqJquery/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html ng-app="test" ng-jq="jQuery_2_1_0">
+  <body>
+      {{jqueryVersion}}
+
+      <script src="../../../../bower_components/jquery/dist/jquery.js"></script>
+      <script src="http://ajax.googleapis.com/ajax/libs/jquery/2.1.0/jquery.min.js"></script>
```

Comment:
We should switch the order of these two jquery script lines because the last loaded jquery wins. If we dropped the `ng-jq` attribute in this app, it would still pass the e2e test.